### PR TITLE
Add telegram notifications

### PR DESCRIPTION
**Pull request by piitaya**
Allow usage of telegram for notifications.

telegram notifications

telegram message

### CODE REVIEW

File: admin/src/page/settings/general.js [hunk 1]
```diff
@@ -522,6 +580,77 @@ class General extends React.Component {
             Test
           </button>
         </section>
+        <section>
+          <p className="main-title mb--2">Telegram</p>
+          <p className="description">
+            Please enter here your telegram chat config. You can find
```
Comment (angrycuban13) [inline admin/src/page/settings/general.js:586]: Capitalize `telegram`

File: admin/src/page/settings/general.js [hunk 2]
```diff
@@ -522,6 +580,77 @@ class General extends React.Component {
...
+            documentation about telegram bot on{" "}
```
Comment (angrycuban13) [inline admin/src/page/settings/general.js:587]: Capitalize `telegram` and add an `'s` to the end of the word

File: admin/src/page/settings/general.js [hunk 3]
```diff
@@ -522,6 +580,77 @@ class General extends React.Component {
...
+            <a
+              href="https://core.telegram.org/bots"
+              target="_blank"
+              rel="noreferrer"
+            >
+              telegram documentation
```
Comment (angrycuban13) [inline admin/src/page/settings/general.js:593]: Same as the previous 2 comments

**Review by angrycuban13 — CHANGES_REQUESTED** [review]
I'm just nitpicking at the UI stuff but @AshDyson will have the fnal voice on this.

File: admin/src/page/settings/general.js (hunk 1 above)
Comment (piitaya) [reply admin/src/page/settings/general.js]: Done 😊

**Review by AshDyson — APPROVED** [review]

**Comment by VozDeOuro** [comment]
I would like to give a suggestion to add the notifications in this style 
image

**Pull request merged**